test/integration/data/output/jobs/super/duper/super-duper-release-3.11-postsubmits.yaml

```diff
@@ -6,6 +6,7 @@ postsubmits:
     decorate: true
     labels:
       artifacts: images
+      ci-operator.openshift.io/variant: ""
```

Comment:
Can we treat a missing label the same way as empty? I don't want to see this add a linearly scaling number of lines to our config ...

Reply:
we could, but the empty value is a lot nicer for people to integrate with.  it's up to you.  I think about all the places where someone wants to find and replace etc.

Reply:
I don't see how it's easier to integrate -- if you're opting in you're adding a line anyway right?

Reply:
I can remove it, makes the change smaller